### PR TITLE
fix: correct client code in the example of TLS connection

### PR DESCRIPTION
**Pull request by iamharvey**
#### Description (what this PR does / why we need it):

A public certificate should be loaded instead of using the server certificate. It could be a root CA cert or an intermediate CA cert. The example may mislead those who are not familiar with TLS practice.

#### Which issue(s) this PR fixes:

fixes #

### CODE REVIEW

**Comment by longXboy** [comment]
Please replace <> with the content of the CA certificate actually generated, which is the last paragraph in server.crt

File: examples/tls/cert/ca.crt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-----BEGIN CERTIFICATE-----
+<pem content>
```
Comment (longXboy) [inline examples/tls/cert/ca.crt:2]: Please replace <pem content> with the content of the CA certificate actually generated, which is the last paragraph in server.crt

**Comment by iamharvey** [comment]
@longXboy fixed

File: examples/tls/cert/ca.crt (hunk 1 above)
Comment (iamharvey) [reply examples/tls/cert/ca.crt]: fixed

**Review by longXboy — APPROVED** [review]

**Pull request merged**